Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/data.py

```diff
@@ -49,6 +49,7 @@
     dict(
         title='Example issue 1',
         timestamp=1639536160,
+        modified=1710884407,
```

Comment:
This is totally fine, although I might suggest making this number a little closer to the timestamp of the bug.

Reply:
Maybe silly, but for posterity, I always tend to use the current timestamp in my code when writing mock data.